values.yaml

```diff
@@ -12,6 +17,9 @@ redis:
   url: ""
 stage: "enterprise"
 ssl:
+  tls:
+    enabled: false
```

Comment:
Same as above, shouldn't the default be `true` for backwards compatibility?

Reply:
Basically only the `Ingress` use this value, and it adds the `tls` spec configurations only whent he `ssl.generator == cert-manager` (on upstream).
Thus, it isn't enabled by default